### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rue-jangel**
hi test, hi this is hi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,6 @@
 Learning about Github
 
 Hello my name is Joelle and I am learning about how Github works!
+
+
+Hi! These are new edits! yay!!!
```
Comment (rue-kevinquigley) [inline README.md]: too many exclamation points for the professional environment

**Review by rue-kevinquigley — CHANGES_REQUESTED** [review]
Almost there. Just a few edits.

File: README.md (hunk 1 above)
Comment (rue-jangel) [reply README.md]: Great point boss, I will update that. Sorry for my lack of professionalism

**Review by rue-kevinquigley — APPROVED** [review]

**Comment by rue-jangel** [comment]
Thanks for approving the changes.

**Pull request merged**